features/facebookImport/src/model/entities/facebook-account.js

```diff
@@ -98,7 +96,7 @@ class FacebookAccount {
 
     forEachOffFacebookEvent(callback) {
         for (const offFacebookCompany of this.offFacebookCompanies) {
-            for (const offFacebookEvent of offFacebookCompany?.events) {
+            for (const offFacebookEvent of offFacebookCompany.events) {
```

Comment:
Guess we don't have to change semantics now, but this is gonna fail before the same as after: The `for ... of` loop here wants an iterable object, a safe way to do this if `offFacebookCompany` could be `null` would be: `offFacebookCompany?.events || []`